templates/code/searchresults.tmpl

```diff
@@ -25,14 +25,14 @@
 				<div class="file-body file-code code-view">
 					<table>
 						<tbody>
+							{{range $k, $line := .Lines}}
 							<tr>
 								<td class="lines-num">
-									{{range .LineNumbers}}
-										<a href="{{$repo.Link}}/src/commit/{{$result.CommitID | PathEscape}}/{{$result.Filename | PathEscapeSegments}}#L{{.}}"><span>{{.}}</span></a>
-									{{end}}
+									<a href="{{$repo.Link}}/src/commit/{{$result.CommitID | PathEscape}}/{{$result.Filename | PathEscapeSegments}}#L{{$k}}"><span>{{$k}}</span></a>
 								</td>
-								<td class="lines-code chroma"><code class="code-inner">{{.FormattedLines}}</code></td>
+								<td class="lines-code chroma"><code class="code-inner">{{$line}}</code></td>
 							</tr>
+							{{end}}
 						</tbody>
```

Comment:
This block sounds like it should be a subtemplate instead as it exists twice.

Reply:
Done.